TodoList.Web/Views/Shared/_LoginPartial.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@using Microsoft.AspNetCore.Identity
+
+@inject SignInManager<IdentityUser> SignInManager
+@inject UserManager<IdentityUser> UserManager
```

Comment:
Avoid using SignInManager and UserManager in your code if possible.

IsSignedIn you can get from HttpContext
Username from ICurrentUser